helm/hpcc/templates/_helpers.tpl

```diff
@@ -615,3 +615,41 @@ resources:
     memory: "100M"
 {{- end -}}
 
+
+{{/*
+Create placement related settings
+*/}}
+{{- define "hpcc.doPlacement" -}}
+{{- if .placement.nodeSelector }}
+nodeSelector:
+{{ toYaml .placement.nodeSelector | indent 2  }}
+{{- end -}}
+{{- if .placement.tolerations }}
+tolerations:
+{{ toYaml .placement.tolerations }}
+{{- end -}}
+{{- if .placement.affinity }}
+affinity:
+{{ toYaml .placement.affinity }}
+{{- end -}}
+{{- if .placement.schedulerNames -}}
+{{- range $profileName := .placement.schedulerNames }}
+schedulerName: {{ $profileName }}
+{{- end -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Check if there is any placement configuration
```

Comment:
it's good to add a comment here, as with other helper functions,
to indicate what's expected to be passed in in dict, e.g.:
Pass in dict with pod and group